zulipterminal/ui_tools/views.py

```diff
@@ -982,12 +993,16 @@ def keypress(self, size: urwid_Size, key: str) -> str:
 class StreamInfoView(PopUpView):
     def __init__(self, controller: Any, color: str,
                  desc: str, title: str) -> None:
+        stream_info_content = [('', [desc])]
+        _, column_widths = self.calculate_table_widths(
```

Comment:
Could we accept the `popup_width` from here and other widths to it later in the subsequent commits? That would avoid the calculation that you're doing in https://github.com/zulip/zulip-terminal/pull/695/commits/89e53d41bc190524e256aeac06fffed1446a4842#diff-5b93264509b3ec67b9370b2bcbaa7798R1002.

Reply:
Yeah, that sounds like a good idea, updated :+1: